src/pages/EventManagement/EventCalender.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from "react";
+import map from "lodash/map";
+import { getEventdata } from "../../api";
+import format from "date-fns/format";
+import Modal from "../../components/Modal";
+
+const EventCalender = () => {
+  const [data, setData] = useState([]);
+  const [isOpenModal, setOpenModal] = useState(false);
+  const [isModalValue, setModalValue] = useState({});
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  const getData = () => {
+    getEventdata().then((res) => {
+      setData(res);
+    });
+  };
+
+  const timeSlots = [
```

Comment:
avoid defining static values inside the component. Create a config.ts in EventMangement directory and define the timeslots in that file.

Reply:
done